components/searchwithin/metadata/searchwithin.yml

```diff
@@ -1,22 +1,7 @@
 name: Search within
+status: Deprecated
+deprecationNotice: Use a [search field](search.html) with a separate control to filter the search instead.
 description: Override the width of the component where necessary.
-sections:
-  - name: Migration Guide
```

Comment:
Should I add deleting migration guides from the docs to the deprecation steps? Or is this something we decide to do ad-hoc?